src/fsharp/FSharp.Core/string.fs

```diff
@@ -104,7 +107,5 @@ namespace Microsoft.FSharp.Core
 
         [<CompiledName("Length")>]
         let length (str:string) =
-            if String.IsNullOrEmpty str then
-                0
-            else
-                str.Length
+            if isNull str then 0
```

Comment:
This is unrelated

Reply:
Well, according to the pr text this is the related change and the stuff in map is unrelated

Reply:
@forki Indeed, sorry for the confusion, will fix that (if I manage to get VS to switch between PRs).